index.d.ts

```diff
@@ -0,0 +1,40 @@
+import * as Promise from 'bluebird';
+import TypedError = require('typed-error');
+
+declare module 'bluebird-lru-cache' {
+
+	export class NoSuchKeyError extends TypedError {
+		public constructor(
+			public key: string,
+		) {
+			super();
```

Comment:
Doesn't having this change this from a type declarations file into a standard typescript file (that just happens to have `.d.ts` instead of just `.ts`)?